src/components/accessibility/screen_reader_only/screen_reader_only.styles.ts

```diff
@@ -28,3 +30,18 @@ export const euiScreenReaderOnly = () => `
   // Chrome requires the negative margin to not cause overflows of parent containers
   margin: -1px;
 `;
+
+/*
+ * Styles
+ */
+export const euiScreenReaderOnlyStyles = (showOnFocus?: boolean) => {
+  if (showOnFocus) {
+    return css`
+      // The :active selector is necessary for Safari which removes :focus when a button is pressed
+      &:not(:focus):not(:active) {
+        ${euiScreenReaderOnly()}
+      }
+    `;
+  }
+  return css(euiScreenReaderOnly());
+};
```

Comment:
IMO this isn't rendering the expected className - it's outputting `emotion-euiScreenReaderOnlyStyles` vs `emotion-euiScreenReaderOnly`. If you return an object format instead of plain `css` I think that should fix the class naming.

```suggestion
export const euiScreenReaderOnlyStyles = (showOnFocus?: boolean) => ({
  euiScreenReaderOnly: showOnFocus
    ? css`
      // The :active selector is necessary for Safari which removes :focus when a button is pressed
      &:not(:focus):not(:active) {
        ${euiScreenReaderOnly()}
      }`
    : css(euiScreenReaderOnly()),
});
```

(sorry in advance for the resulting snapshot thrashing 🙃)